lms/templates/main.html

```diff
@@ -182,3 +189,27 @@
     )
   ) if course_id and enrollment_action else ""
 }</%def>
+
+# Performance beacon for onload times
+% if settings.FEATURES.get('ENABLE_ONLOAD_BEACON', False):
+<script>
+  sample_rate = ${settings.ONLOAD_BEACON_SAMPLE_RATE}
+  roll = Math.floor(Math.random() * 100)/100
+
+  if(roll < sample_rate){
+    onloadBeaconSent = false;
+    $(window).load(function() {
+      var data = {
+        event: "onload",
+        value: Date.now() - loadStartTime,
```

Comment:
What is the actual measurement you're looking for?  Time from navigationStart to loadEventEnd?